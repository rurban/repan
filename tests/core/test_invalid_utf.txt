# Test invalid UTF data

/%7f_%c1%bf/u
    print

/%c2%80_%c1%bf/u
    print

/%c2%80_%c2/u
    print

/%c2%80_%c2___/u
    print

/%df%bf_%e0%a0%80_%e0%9f%bf/u
    print

/%ed%9f%bf_%ed%a0%80/u
    print

/%ee%80%80_%ed%bf%bf/u
    print

/%ef%bf%bf_%ef%bf/u
    print

/%ef%bf%bf_%ef%bf___/u
    print

/%f4%8f%bf%bf_%f4%90%80%80/u
    print

/%f0%90%80%80_%f0%8f%bf%bf/u
    print

/%f0%90%80%80_%f0%90/u
    print

/%f0%90%80%80_%f0%90___/u
    print

/%d7ff%d800/u16
    print

/%ffff%d800%d800/u16
    print

/%e000%dfff%d800%dc00/u16
    print

/%e000%dbff%dfff%d800%dc00%dc00/u16
    print
