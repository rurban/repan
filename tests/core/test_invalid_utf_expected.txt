# Test invalid UTF data

/%7f_%c1%bf/u
    print
    Pattern parse failed: Invalid UTF character
        /%7f_[ HERE ]>>%c1%bf/

/%c2%80_%c1%bf/u
    print
    Pattern parse failed: Invalid UTF character
        /%c2%80_[ HERE ]>>%c1%bf/

/%c2%80_%c2/u
    print
    Pattern parse failed: Invalid UTF character
        /%c2%80_[ HERE ]>>%c2/

/%c2%80_%c2___/u
    print
    Pattern parse failed: Invalid UTF character
        /%c2%80_[ HERE ]>>%c2___/

/%df%bf_%e0%a0%80_%e0%9f%bf/u
    print
    Pattern parse failed: Invalid UTF character
        /%df%bf_%e0%a0%80_[ HERE ]>>%e0%9f%bf/

/%ed%9f%bf_%ed%a0%80/u
    print
    Pattern parse failed: Invalid UTF character
        /%ed%9f%bf_[ HERE ]>>%ed%a0%80/

/%ee%80%80_%ed%bf%bf/u
    print
    Pattern parse failed: Invalid UTF character
        /%ee%80%80_[ HERE ]>>%ed%bf%bf/

/%ef%bf%bf_%ef%bf/u
    print
    Pattern parse failed: Invalid UTF character
        /%ef%bf%bf_[ HERE ]>>%ef%bf/

/%ef%bf%bf_%ef%bf___/u
    print
    Pattern parse failed: Invalid UTF character
        /%ef%bf%bf_[ HERE ]>>%ef%bf___/

/%f4%8f%bf%bf_%f4%90%80%80/u
    print
    Pattern parse failed: Invalid UTF character
        /%f4%8f%bf%bf_[ HERE ]>>%f4%90%80%80/

/%f0%90%80%80_%f0%8f%bf%bf/u
    print
    Pattern parse failed: Invalid UTF character
        /%f0%90%80%80_[ HERE ]>>%f0%8f%bf%bf/

/%f0%90%80%80_%f0%90/u
    print
    Pattern parse failed: Invalid UTF character
        /%f0%90%80%80_[ HERE ]>>%f0%90/

/%f0%90%80%80_%f0%90___/u
    print
    Pattern parse failed: Invalid UTF character
        /%f0%90%80%80_[ HERE ]>>%f0%90___/

/%d7ff%d800/u16
    print
    Pattern parse failed: Invalid UTF character
        /%d7ff[ HERE ]>>%d800/

/%ffff%d800%d800/u16
    print
    Pattern parse failed: Invalid UTF character
        /%ffff[ HERE ]>>%d800%d800/

/%e000%dfff%d800%dc00/u16
    print
    Pattern parse failed: Invalid UTF character
        /%e000[ HERE ]>>%dfff%d800%dc00/

/%e000%dbff%dfff%d800%dc00%dc00/u16
    print
    Pattern parse failed: Invalid UTF character
        /%e000%dbff%dfff%d800%dc00[ HERE ]>>%dc00/
